### PR TITLE
de-DE: Apply #674, #693, #698

### PR DESCRIPTION
**Pull request by dVoCex**
Added STR_5666, STR_5861 - STR_5870

### CODE REVIEW

File: data/language/de-DE.txt [hunk 1]
```diff
@@ -4159,6 +4160,16 @@ STR_5856    :{SMALLFONT}{BLACK}Die im Vollbildmodus verwendete{NEWLINE}Auflösun
 STR_5857    :{SMALLFONT}{BLACK}Spieloptionen
 STR_5858    :{SMALLFONT}{BLACK}GPU anstelle der CPU für die Anzeige verwenden. Verbessert die Kompatibilität mit Aufnahmeprogrammen. Möglicherweise wird die Leistung geringfügig beeinträchtigt
 STR_5859    :{SMALLFONT}{BLACK}Aktiviert Tweening für ein sichtbar flüssigeres Spielverhalten. Wenn deaktiviert läuft das Spiel mit 40 FPS
+STR_5861    :Schlüssel Überprüfung fehlgeschlagen.
```
Comment (Gymnasiast) [inline data/language/de-DE.txt]: Compound words shouldn't have a space in German. This should read `Schlüsselüberprüfung`. This also applies to many other lines.

Comment (dVoCex) [inline data/language/de-DE.txt]: You're right. I'll fix it as soon as I am at home.

File: data/language/de-DE.txt [hunk 2]
```diff
@@ -4159,6 +4160,16 @@ STR_5856    :{SMALLFONT}{BLACK}Die im Vollbildmodus verwendete{NEWLINE}Auflösun
 STR_5857    :{SMALLFONT}{BLACK}Spieloptionen
 STR_5858    :{SMALLFONT}{BLACK}GPU anstelle der CPU für die Anzeige verwenden. Verbessert die Kompatibilität mit Aufnahmeprogrammen. Möglicherweise wird die Leistung geringfügig beeinträchtigt
 STR_5859    :{SMALLFONT}{BLACK}Aktiviert Tweening für ein sichtbar flüssigeres Spielverhalten. Wenn deaktiviert läuft das Spiel mit 40 FPS
+STR_5861    :Schlüsselüberprüfung fehlgeschlagen.
+STR_5862    :Blockiere unbekannte Spieler.
+STR_5863    :{SMALLFONT}{BLACK}Erlaube nur Spieler, mit bekanntem Schlüssel, sich zu verbinden.
+STR_5864    :Dieser Server erlaubt nur Spieler, die auf der Whitelist stehen.
+STR_5865    :Logge Chat Verlauf
```
Comment (Gymnasiast) [inline data/language/de-DE.txt]: *Chatverlauf.

There are some other compound words in the following string you still need to correct.

File: data/language/de-DE.txt [hunk 3]
```diff
@@ -4159,6 +4160,16 @@ STR_5856    :{SMALLFONT}{BLACK}Die im Vollbildmodus verwendete{NEWLINE}Auflösun
...
+STR_5866    :{SMALLFONT}{BLACK}Speichert den gesamten Chat Verlauf in deinem User Verzeichnis.
```
Comment (Gymnasiast) [inline data/language/de-DE.txt]: *Userverzechnis. Or probably just Benutzerverzeichnis.

**Comment by dVoCex** [comment]
@Gymnasiast I have improved the translation, thank you for your hints.

File: data/language/de-DE.txt [hunk 4]
```diff
@@ -4159,6 +4160,16 @@ STR_5856    :{SMALLFONT}{BLACK}Die im Vollbildmodus verwendete{NEWLINE}Auflösun
 STR_5857    :{SMALLFONT}{BLACK}Spieloptionen
 STR_5858    :{SMALLFONT}{BLACK}GPU anstelle der CPU für die Anzeige verwenden. Verbessert die Kompatibilität mit Aufnahmeprogrammen. Möglicherweise wird die Leistung geringfügig beeinträchtigt
 STR_5859    :{SMALLFONT}{BLACK}Aktiviert Tweening für ein sichtbar flüssigeres Spielverhalten. Wenn deaktiviert läuft das Spiel mit 40 FPS
+STR_5861    :Schlüsselüberprüfung fehlgeschlagen.
+STR_5862    :Blockiere unbekannte Spieler.
+STR_5863    :{SMALLFONT}{BLACK}Erlaube nur Spieler, mit bekanntem Schlüssel, sich zu verbinden.
+STR_5864    :Dieser Server erlaubt nur Spieler,{NEWLINE}die auf der Whitelist / Positivliste stehen.
+STR_5865    :Logge Chatverlauf
+STR_5866    :{SMALLFONT}{BLACK}Speichert den gesamten Chatverlauf in deinem Benutzerverzeichnis.
+STR_5867    :{WINDOW_COLOUR_2}Provider Name: {BLACK}{STRING}
```
Comment (Gymnasiast) [inline data/language/de-DE.txt]: These three strings (5867, 5868, 5869) still need correcting.

**Comment by Nubbie** [comment]
This PR will close #674 #693 and #698 :+1:

**Pull request merged**